### ISSUE TITLE
Playground doesnt work

### ISSUE THREAD

**Issue by gitihobo** [riffusion/riffusion#152]
How can I get it to work? It doesnt seem to have a model it needs maybe? What can I troubleshoot

**Comment by rsxdalv** [comment]
Could it be this problem: https://github.com/riffusion/riffusion/issues/148 or an installation issue?

**Comment by gitihobo** [comment]
Yes first I get A:\\anaconda3\envs\riffusion\riffusion\riffusion\riffusion_pipeline.py:14: FutureWarning: Importing `DiffusionPipeline` or `ImagePipelineOutput` from diffusers.pipeline_utils is deprecated. Please import from diffusers.pipelines.pipeline_utils instead.
  from diffusers.pipeline_utils import DiffusionPipeline
text_encoder\model.safetensors not found
You have passed a non-standard module <function load_stable_diffusion_pipeline.<locals>.<lambda> at 0x00000129EE22AEE0>. We cannot verify whether it has the correct type
A:\\anaconda3\envs\riffusion\lib\site-packages\diffusers\pipelines\stable_diffusion\pipeline_stable_diffusion.py:181: FutureWarning: The configuration file of the unet has set the default `sample_size` to smaller than 64 which seems highly unlikely. If your checkpoint is a fine-tuned version of any of the following:
- CompVis/stable-diffusion-v1-4
- CompVis/stable-diffusion-v1-3
- CompVis/stable-diffusion-v1-2
- CompVis/stable-diffusion-v1-1
- runwayml/stable-diffusion-v1-5
- runwayml/stable-diffusion-inpainting
 you should change 'sample_size' to 64 in the configuration file. Please make sure to update the config accordingly as leaving `sample_size=32` in the config might lead to incorrect results in future versions. If you have downloaded this checkpoint from the Hugging Face Hub, it would be very nice if you could open a Pull request for the `unet/config.json` file
 
 then 
 
 TypeError: 'bool' object is not iterable
Traceback:
File "A:\\anaconda3\envs\riffusion\lib\site-packages\streamlit\runtime\scriptrunner\script_runner.py", line 552, in _run_script
    exec(code, module.__dict__)
File "A:\\anaconda3\envs\riffusion\riffusion\riffusion\streamlit\playground.py", line 34, in <module>
    render()
File "A:\\anaconda3\envs\riffusion\riffusion\riffusion\streamlit\playground.py", line 29, in render
    module.render()
File "A:\\anaconda3\envs\riffusion\riffusion\riffusion\streamlit\tasks\text_to_audio.py", line 95, in render
    image = streamlit_util.run_txt2img(
File "A:\\anaconda3\envs\riffusion\lib\site-packages\streamlit\runtime\caching\cache_utils.py", line 211, in wrapper
    return cached_func(*args, **kwargs)
File "A:\\anaconda3\envs\riffusion\lib\site-packages\streamlit\runtime\caching\cache_utils.py", line 240, in __call__
    return self._get_or_create_cached_value(args, kwargs)
File "A:\\anaconda3\envs\riffusion\lib\site-packages\streamlit\runtime\caching\cache_utils.py", line 266, in _get_or_create_cached_value
    return self._handle_cache_miss(cache, value_key, func_args, func_kwargs)
File "A:\\anaconda3\envs\riffusion\lib\site-packages\streamlit\runtime\caching\cache_utils.py", line 320, in _handle_cache_miss
    computed_value = self._info.func(*func_args, **func_kwargs)
File "A:\\anaconda3\envs\riffusion\riffusion\riffusion\streamlit\util.py", line 174, in run_txt2img
    output = pipeline(
File "A:\\anaconda3\envs\riffusion\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
File "A:\\anaconda3\envs\riffusion\lib\site-packages\diffusers\pipelines\stable_diffusion\pipeline_stable_diffusion.py", line 764, in __call__
    do_denormalize = [not has_nsfw for has_nsfw in has_nsfw_concept]

**Comment by imVinayPandya** [comment]
same error
`TypeError: 'bool' object is not iterable`

**Comment by justanothernguyen** [comment]
This is an issue with not pinning dependencies. If you recently installed riffussion, chances are you have a version with breaking changes.

Try changing the requirements.txt

Instead of

`diffusers>=0.9.0`

Pin it to 0.9.0:

`diffusers=0.9.0`

I had issues with torch , torchaudio and torchvision too because there's a breaking change on 2.1.x (released recently), have to pin those to 2.0.1

**Comment by davidmartinrius** [comment]
> This is an issue with not pinning dependencies. If you recently installed riffussion, chances are you have a version with breaking changes.
> 
> Try changing the requirements.txt
> 
> Instead of
> 
> `diffusers>=0.9.0`
> 
> Pin it to 0.9.0:
> 
> `diffusers=0.9.0`
> 
> I had issues with torch , torchaudio and torchvision too because there's a breaking change on 2.1.x (released recently), have to pin those to 2.0.1

Hello everyone!

I followed your steps and it worked. For anyone that needs the commands to fix a fresh installation and already installed the requirements.txt and supposing you are working with NVIDIA GPU:

pip uninstall diffusers
pip install diffusers==0.9.0

pip uninstall torch torchvision torchaudio
pip install torch==2.0.1+cu118 torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118

Thank you @justanothernguyen !

David Martin Rius
